Sources/Jobs/MemoryJobQueue.swift

```diff
@@ -46,8 +46,8 @@ public final class MemoryQueue: JobQueueDriver {
     ///   - job: Job
     ///   - eventLoop: Eventloop to run process on (ignored in this case)
     /// - Returns: Queued job
-    @discardableResult public func push(_ buffer: ByteBuffer) async throws -> JobID {
-        return try await self.queue.push(buffer)
+    @discardableResult public func push(_ buffer: ByteBuffer, options: JobExecutionOptions) async throws -> JobID {
+        return try await self.queue.push(buffer, executionOptions: options)
```

Comment:
Can we propagate the `executionOptions` -> `options` change through all of the functions. Including the type name, should be `JobOptions`.